client/my-sites/backup/backup-contents-page/style.scss

```diff
@@ -11,5 +11,21 @@
 
 	&__body {
 		border-top: 1px solid var(--studio-gray-5);
+		padding-top: 30px;
+	}
+
+	.file-browser-node {
+		&__loading {
+			&.placeholder {
+				height: 32px;
```

Comment:
```suggestion
				height: 16px;
				margin: 8px;
```

Reply:
I think this looks a little better, but might want to get design input regardless.

Reply:
Looks good! Just updated it :)

Reply:
I will take notes on all parts of the flow that might require loading placeholders and can ask design input for further iterations.